CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ENHANCEMENTS:
 * config: Terraform can now track some additional detail about values that won't be known until the apply step, such as the range of possible lengths for a collection or whether an unknown value can possibly be null. When this information is available, Terraform can potentially generate known results for some operations on unknown values. This doesn't mean that Terraform can immediately track that detail in all cases, but the type system now contains the facility for that and so over time we will improve the level of detail generated by built-in functions, language operators, Terraform providers, etc. [GH-33234]
 * jsonplan: Added `errored` field to JSON plan output, indicating whether a plan errored. [GH-33372]
+* cloud: Terraform now supports setting the Terraform Cloud project via the cloud block and `TF_CLOUD_PROJECT` environment variable. When a cloud configuration creates a new workspace it will be created in the given project. If the project does not exist it will be created. `terraform workspace list` will also respect listing matching workspaces in the configured project.
```

Comment:
Drive-by note: we usually make changelog updates as a separate step after a PR is merged in order to avoid being trapped in merge conflict purgatory!

Reply:
Removed this